docs/core/tools/dotnet-install-script.md

```diff
@@ -30,76 +28,102 @@ We recommend that you use the stable version that is hosted on [.NET Core main w
 
-Specifies the installation path. The directory is created if it doesn't exist. The default value is *%LocalAppData%\.dotnet*. Note that binaries are placed directly in the directory.
+Specifies the installation path. The directory is created if it doesn't exist. The default value is *%LocalAppData%\.dotnet*. Binaries are placed directly in the directory.
 
 `-Architecture <ARCHITECTURE>`
 
 Architecture of the .NET Core binaries to install. Possible values are `auto`, `x64`, and `x86`. The default value is `auto`, which represents the currently running OS architecture.
 
 `-SharedRuntime`
 
-If set, this switch limits installation to the shared runtime. The entire SDK isn't installed.
+> [!NOTE]
+> This parameter is obsolete and may be removed in a future version of the script. The recommended alternative is the `Runtime` option.
+
+Installs just the shared runtime bits, not the entire SDK. This is equivalent to specifying `-Runtime dotnet`.
+
+`-Runtime <RUNTIME>`
+
+Installs just the shared runtime, not the entire SDK. The possible values are:
+
+* `dotnet` - the `Microsoft.NETCore.App` shared runtime.
+* `aspnetcore` - the `Microsoft.AspNetCore.App` shared runtime.
 
 `-DryRun`
 
-If set, the script won't perform the installation; but instead, it displays what command line to use to consistently install the currently requested version of the .NET Core CLI. For example if you specify version `latest`, it displays a link with the specific version so that this command can be used deterministically in a build script. It also displays the binary's location if you prefer to install or download it yourself.
+If set, the script won't perform the installation. Instead, it displays what command line to use to consistently install the currently requested version of the .NET Core CLI. For example, if you specify version `latest`, it displays a link with the specific version so that this command can be used deterministically in a build script. It also displays the binary's location if you prefer to install or download it yourself.
 
 `-NoPath`
 
-If set, the prefix/installdir are not exported to the path for the current session. By default, the script will modify the PATH, which makes the CLI tools available immediately after install.
+If set, the installation folder isn't exported to the path for the current session. By default, the script modifies the PATH, which makes the CLI tools available immediately after install.
+
+`-Verbose`
+
+Displays diagnostics information.
 
 `-AzureFeed`
 
-Specifies the URL for the Azure feed to the installer. It isn't recommended that you change this value. The default is `https://dotnetcli.azureedge.net/dotnet`.
+Specifies the URL for the Azure feed to the installer. It isn't recommended that you change this value. The default value is `https://dotnetcli.azureedge.net/dotnet`.
+
+`-UncachedFeed`
+
+It allows changing the URL for the Uncached feed used by this installer. It isn't recommended that you change this value.
```

Comment:
It allows --> Allows
nit: It isn't recommended that you --> We recommend that you don't